mcs/class/Mono.Posix/Mono.Posix-netstandard_2_0.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <AssemblyName>Mono.Posix</AssemblyName>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <!--<BaseIntermediateOutputPath>obj-netstandard2.0</BaseIntermediateOutputPath>
+    <IntermediateOutputPath>obj-netstandard2.0</IntermediateOutputPath>-->
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(ForceUseLibC)' == 'true'">
+    <DefineConstants>FORCE_USE_LIBC_NOT_MSVC</DefineConstants>

```

Comment:
I think you should use:

`<DefineConstants>$(DefineConstants);FORCE_USE_LIBC_NOT_MSVC</DefineConstants>`

because there are define constants in the default SDK.props that you will be overwriting.  For example:

https://github.com/dotnet/sdk/blob/16b2d88ed0f936734bdf042359de55bd4015591d/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.CSharp.props#L21